src/share/dropup-button.js

```diff
@@ -1,84 +1,98 @@
 // @flow
-import React, {PropTypes as t} from 'react';
+import React, {PropTypes as t, Component} from 'react';
```

Comment:
A very minor thing, but I think we should alphabetize imports just to make it quicker to spot a particular one ... Component before PropTypes.